cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/TopologyExport.java

```diff
@@ -133,23 +133,21 @@ private static String findOrCreateTopologicalNode(VoltageLevel vl, int node) {
 
         Optional<Bus> selectedBus = nodeSet.stream().map(n -> getBusForBusBreakerViewBus(vl, n)).filter(Objects::nonNull).findFirst();
         if (selectedBus.isPresent()) {
-            return selectedBus.get().getId();
-        }
-
-        Optional<Integer> selectedNode = nodeSet.stream().sorted().findFirst();
-        if (selectedNode.isEmpty()) {
-            throw new PowsyblException("nodeSet never can be empty");
+            return context.getNamingStrategy().getCgmesId(selectedBus.get());
         }
 
-        return vl.getId() + "_" + selectedNode.get();
+        return context.getNamingStrategy().getCgmesId(nodeSet.stream()
+                .sorted()
+                .findFirst()
+                .map(selectedNode -> "DISCONNECTED_" + vl.getId() + "_" + selectedNode)
```

Comment:
At least some of the topological nodes created as "disconnected" miss the terminal definition in the EQ file. I'm checking it on real-world test cases.

Reply:
Not sure why, the terminals are created after in `writeSwitchTerminal`, no?

Reply:
Topological nodes and terminals should not be written for fictitious switches. `context.isExportedEquipment(sw)` is `false`, we do not have to export TN definitions and terminal references.

Reply:
The same in SSH export.

Reply:
If we remove the `DISCONNECTED_` prefix, the TNs will watch the ones used in SV export